include/ppx/knob.h

```diff
@@ -69,11 +69,18 @@ class Knob
 
     virtual void ResetToDefault() = 0;
 
+    // Returns true if there has been an update to the knob value and lowers mUpdatedFlag
```

Comment:
nit: remove the mUpdatedFlag form the comment. Comment here should be about usage. mUpdatedFlag is an internal implementation detail.